meta/template.md

```diff
@@ -1,48 +1,38 @@
 
-# Spec Name
+# Title: A human-readable title for this proposal
 
-* Status: In Review / Reviewed / Implementing / Implemented
-* Author(s): [Author Name](https://github.com/NuGet)
-* Issue: [1](https://github.com/NuGet/Home/issues/1) Issue Title
+[Author Name](https://github.com/<username>) | [GitHub Issue](https://github.com/NuGet/Home/issues/<issue>)
 
-## Problem Background
+## Description: What is it?
 
-Present the background and the motivation for this design. 
-Present evidence why this is an important problem to tackle.
+A concise, one-paragraph description of the proposal.
 
-## Who are the customers
+## Problem: What problem is this solving?
 
-Customers affected by this change.
+A short background of the problem and how your solution will solve it.
 
-## Goals
+## Why: How do we know this is a real problem and worth solving?
 
-Specifically call out the goals for this proposed design.
+A few details on why we're doing this, and what pain points this solves. Use real, concrete examples to make your case.
 
-## Non-Goals
+## Success: How do we know if we've solved this problem?
 
-Call out things that are not in consideration for this design. 
+A couple ways on how we might measure the success or outcome of this proposal. 
 
-## Solution Overview
+## Audience: Who are we building for?
 
-Describe the proposed solution in as much detail as you deem necessary.
+A brief description to help everyone understand who this proposal is for.
 
-## Test Strategy
+## What: What does this look like in the product?
 
-Other than Unit Tests, which automated and manual integration, component, and end-to-end tests will be needed to validate the implementation? Is a special roll-out strategy (including kill-switch, opt-in flag, etc.) needed?
+A low-fidelity mockup or pseduocode that shows how this proposal would look.
 
-## Future Work
+## How: How should it be implemented?
 
-Declare any future work if applicable.
+A design or implementation for bringing the proposal to life. 
 
-## Open Questions
+## When: When does this ship and what are the milestones?
 
-Any open questions not specifically called out in the design.
+A timeline including ideal dates to ship the proposal & any key milestones (if any).
 
-## Considerations
 
-Any other solutions or approaches considered? 
-The rejected designs are often just as important as the proposed ones!
-
-### References
-
-References to related issues, features
```

Comment:
What do you think of keeping the "References" section? In my mind, I want to keep the previous sections to the point so that the reader can follow my thought process. I wouldn't want to overwhelm the reader with many links in the "Problem: ..." or "Why: ..." sections. On the other hand, the "References" section at the bottom is the perfect place to drop miscellaneous links for folks that want to drill deeper on the topic.

Reply:
I looked at other design repos & didn't find any references to be needed.

https://github.com/rust-lang/rfcs/tree/master/text
https://github.com/npm/rfcs/tree/latest/accepted
https://github.com/dotnet/designs/tree/main/accepted/2020

I think the bottom may need a Q&A section though?

Reply:
Also, i just took a stab at the common things needed. Maybe there's a better combination from the various RFC & design processes already established.

Reply:
Q&A would work too. FYI, I don't feel strongly about this feedback so feel free to skip it :)

Reply:
There's a great section here on what type of questions are needed:

http://smallcultfollowing.com/babysteps/blog/2018/06/20/proposal-for-a-staged-rfc-process/#the-proposal-point-and-the-change-from-spitballing-to-designing (Rust RFC process)

My approach was to use a typical 1-pager process for a proposal. If anything is more than 1 page as a proposal, it might be hard to understand.

I think the `Considerations` or `Cross Cutting Concerns` / etc should be a section?

For `References`, that's how I was treating it as well. It would be part of the other areas or in the design itself.

Reply:
Sounds good to me.